vue-custom-bootstrap/src/router/index.ts

```diff
@@ -1,15 +1,26 @@
 import Vue from "vue";
 import VueRouter from "vue-router";
-import Assessment from "../views/Assessment.vue";
+// import Assessment from "../views/Assessment.vue";
+import Profile from "../views/Profile.vue";
 
 Vue.use(VueRouter);
 
 const routes = [
   {
     path: "/",
-    name: "Assessment",
-    component: Assessment
+    name: "Profile",
+    component: Profile
   }
+  // {
```

Comment:
Please delete the dead (commented out) code

Reply:
okay